complete/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+#These are all configurations to connect mySQL to this project
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/daily_stock_prices
+spring.datasource.username=jen
+spring.datasource.password=sideproject123
```

Comment:
This isn't something you have to worry about now, but on a real project, you shouldn't commit login info that isn't encrypted. So just keep that in mind